rpc/client.go

```diff
@@ -58,7 +59,7 @@ var clientRetryOptions = retry.Options{
 
 // init creates a new client RPC cache.
 func init() {
-	clients = map[string]*Client{}
+	clients = make(map[string]*Client)
```

Comment:
There is no real reason to prefer `make` over the empty map construction. We use the latter frequently throughout the code. Let's leave this alone.